ostree-compliance-mode.spec

```diff
@@ -0,0 +1,58 @@
+%global dracutdir %(pkg-config --variable=dracutdir dracut)
```

Comment:
This rpm is not related to dracut, if you want some other spec files to reference, here's another resource:

https://gitlab.com/CentOS/automotive/rpms

Line 1 and 2 should be deleted here

Reply:
Agreed, that should have been removed. I'll take that out